src/components/userComponents/emailPopup/page.tsx

```diff
@@ -0,0 +1,198 @@
+              as={Fragment}
+              enter="ease-out duration-300"
+              enterFrom="opacity-0"
+              enterTo="opacity-100"
+              leave="ease-in duration-200"
+              leaveFrom="opacity-100"
+              leaveTo="opacity-0"
+            >
+              <Dialog.Overlay className="fixed inset-0" />
+            </Transition.Child>
+
+            {/* This element is to trick the browser into centering the modal contents. */}
+            <span
+              className="inline-block h-screen align-middle"
+              aria-hidden="true"
+            >
+              &#8203;
+            </span>
+            <Transition.Child
+              as={Fragment}
+              enter="ease-out duration-300"
+              enterFrom="opacity-0 scale-95"
+              enterTo="opacity-100 scale-100"
+              leave="ease-in duration-200"
+              leaveFrom="opacity-100 scale-100"
+              leaveTo="opacity-0 scale-95"
+            >
+            <div className="fixed inset-0 flex items-center justify-center">
+                <div className="w-[334px] h-[404px] bg-[#EBF0E8] rounded-lg shadow-box relative">
+                  
+                  
+
+                    <div className="flex justify-end">
+                      <button
+                        type="button"
+                        className="inline-flex justify-center duration-300 focus:outline-none"
+                  
+                      >
+                      <VscClose 
+                          className="text-4xl text-[#808080] pt-[17px] pr-[17px]"
+                          onClick={closeModal}/>
+                      </button>
+                    </div>
+
+                    {subscribed ? (
+                      <><div className="absolute pt-[52px] top-[72px] left-1/2 transform -translate-x-1/2 -translate-y-1/2">
+                    <PiSealCheck className="text-[#3F6A38] text-5xl" />
+                      </div>
+                  
+                      <h2 className="pt-[108px] pl-[32px] pr-[32px] text-center font-Lato font-bold text-[19px] text-[#3B3B3B]">
+                        THANKS FOR SUBSCRIBING!
+                      </h2>
+                      
+                      <p className="text-center text-sm font-Lato text-[#3B3B3B] pt-[10px] pl-[30px] pr-[30px] text-[16px]">
+                      Your sign-up request was successful! Please check your email inbox to confirm.</p>
+                      
+
+                    <Link href="/collectionsPage" >
+
+                      <div className="flex items-center justify-center pt-[24px]">
+                      <button
+                        type="button"
+                        className="bg-[#7CA24E] w-[283px] h-[43px] text-white rounded-2xl p-[21px] mt-[8px] flex items-center justify-center"
+                      >
+                        Back to Home
+                      </button>
+                    </div>
+                    </Link>
```

Comment:
For readability/maintainability:  Lets define this success page as its own helper component somewhere earlier in this file